Tests/MongoSwiftTests/ClientSessionTests.swift

```diff
@@ -48,4 +48,50 @@ final class ClientSessionTests: MongoSwiftTestCase {
             expect(escapedSession?.active).to(beFalse())
         }
     }
+
+    func testCommandEventStreamClient() async throws {
+        try await self.withTestClient { client in
+            let commandStr: [String] = ["ping", "ping", "endSessions", "endSessions"]
+            let eventTypes: [EventType] = [
+                .commandStartedEvent,
+                .commandSucceededEvent,
+                .commandStartedEvent,
+                .commandSucceededEvent
+            ]
+            Task {
+                var i = 0
+                for try await event in client.commandEvents {
+                    expect(commandStr[i]).to(equal(event.commandName))
+                    expect(eventTypes[i]).to(equal(event.type))
+
+                    i += 1
+                }
+            }
+            try await client.db("admin").runCommand(["ping": 1])
+        }
+    }
+
+    func testSDAMEventStreamClient() async throws {
+        try await self.withTestClient { client in
+            Task {
+                var i = 0
+                var eventHandler: [EventType] = []
```

Comment:
Named it `eventTypes`. Looked into `NioConcurrencyHelpers.Lock` that `APMUtils.swift` uses and I locked the appending to the array (since you cant pass in an async function into the .`withLock{ ... }`) and left associated comments.